### ISSUE TITLE
Warn when closing Qube Manager if a qube restart is in process

### ISSUE THREAD

**Issue by UndeadDevel** [QubesOS/qubes-issues#9078]
### Qubes OS release
4.2

### Brief summary
Qube restart triggered via the Qube Manager will shut down, but not restart a qube if the Qube Manager is closed after pressing the button.

### Steps to reproduce
See summary.

### Expected behavior
When restart is pressed, a restart occurs.

### Actual behavior
Restart only occurs if the Qube Manager instance is kept running.

**Comment by marmarek** [comment]
Yes, this is indeed how restart function in qubes manager works - it's responsible for starting the qube after it shuts off, so if you close the manager in the meantime, nothing will start the qube again. I guess we can add a warning if any qube restart is in the process when you (try to) close the manager.
